server/testutils/data.go

```diff
@@ -102,6 +102,16 @@ func GetUser(role, email string) *model.User {
 	}
 }
 
+func GetRemoteUser(role, email, remoteID string) *model.User {
+	return &model.User{
+		Id:       GetID(),
+		Username: "test-user",
+		Roles:    role,
+		Email:    email,
+		RemoteId: &remoteID,
```

Comment:
For completeness, should we set `EmailVerified` to true for this test user?

Reply:
Hum, I purposely left it to `false` to make sure my code set it to `true` 😅

Reply:
No, that's fine too! Just wanted to clarify -- your call either way :)